traefik/traefik.toml

```diff
@@ -0,0 +1,23 @@
+defaultEntryPoints = ["http", "https"]
+
+[entryPoints]
+  [entryPoints.http]
+  address = ":80"
+    [entryPoints.http.redirect]
+      entryPoint = "https"
+  [entryPoints.https]
+  address = ":443"
+    [entryPoints.https.tls]
+
+[web]
```

Comment:
Web is deprecated. Played with it and with
```
[entrypoints]
  #...
  # https://docs.traefik.io/configuration/api/#authentication
  [entrypoints.webentry]
    address = ":8080"
    [entrypoints.webentry.auth]
      [entrypoints.webentry.auth.basic]
        users = ["admin:encryptedkey"]
  [entryPoints.http]
...........
```
Things work. See https://www.digitalocean.com/community/tutorials/how-to-use-traefik-as-a-reverse-proxy-for-docker-containers-on-ubuntu-16-04?comment=77114